packages/xo-common/src/api-errors.js

```diff
@@ -200,3 +200,10 @@ export const notEnoughResources = create(24, data => ({
   data, // [{ resourceSet, resourceType, available, requested }]
   message: 'not enough resources in resource set',
 }))
+
+export const runningJobs = create(25, ({ ids }) => ({
```

Comment:
I dislike adding new specific API errors, especially this one which does not look like an error at all.

Anyway to improve this? Maybe by using `incorrectState`? https://github.com/vatesfr/xen-orchestra/blob/da8eec06abb1d49c00b65526a384dff64f9a23dc/packages/xo-common/src/api-errors.js#L103-L113